### ISSUE TITLE
The loss func in pytorch version

### ISSUE THREAD

**Issue by InspirationLaurie** [GeoZcx/A-deeply-supervised-image-fusion-network-for-change-detection-in-remote-sensing-images#4]
I found that the loss funcion cannot handle the tuple returned by your model. The code of the PyTorch version may wrong. 

The loss may like the following formats

`
for itr in itr_loss:
    loss +=  F.binary_cross_entropy(input, target)
`

**Comment by GeoZcx** [comment]
The loss function accepts one predict and one target. To get the total loss of the model, you will have to manually collect the loss of each predicted image returned by the model.
